### ISSUE TITLE
Deprecation warning with cryptography 3.4

### ISSUE THREAD

**Issue by serhiy-storchaka** [mpdavis/python-jose#208]
Import from `cryptography.utils` raises now a `cryptography.utils.CryptographyDeprecationWarning` because function `int_from_bytes` is deprecated.
https://github.com/mpdavis/python-jose/blob/master/jose/backends/cryptography_backend.py#L23

It will be an error in future versions, after removing `int_from_bytes`. But even now it breaks tests because `pytest` converts warnings to errors.

**Comment by serhiy-storchaka** [comment]
Example of traceback:
```
    from jose import jwt
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/jose/jwt.py:13: in <module>
    from jose import jws
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/jose/jws.py:11: in <module>
    from jose import jwk
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/jose/jwk.py:9: in <module>
    from jose.backends.base import Key
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/jose/backends/__init__.py:3: in <module>
    from jose.backends.cryptography_backend import CryptographyRSAKey as RSAKey  # noqa: F401
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/jose/backends/cryptography_backend.py:18: in <module>
    from cryptography.utils import int_from_bytes, int_to_bytes
/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/cryptography/utils.py:104: in __getattr__
    warnings.warn(obj.message, obj.warning_class, stacklevel=2)
E   cryptography.utils.CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
```

**Comment by amejias-manager** [comment]
This should be fix in #207

**Comment by brian316** [comment]
Until it is fixed you can do this workaround to get rid of the warning for tests

```python
import warnings
with warnings.catch_warnings():
    # this will suppress all warnings in this block
    warnings.filterwarnings('ignore', message='int_from_bytes is deprecated')
    function_that_raises_warnings()
```

**Comment by Hultner** [comment]
> Until it is fixed you can do this workaround to get rid of the warning for tests
> 
> ```python
> import warnings
> with warnings.catch_warnings():
>     # this will suppress all warnings in this block
>     warnings.filterwarnings('ignore', message='int_from_bytes is deprecated')
>     function_that_raises_warnings()
> ```

I rather keep the warnings as a reminder, but none the less useful for those who doesn't-

**Comment by allanlewis** [comment]
Am I correct that this has been resolved in v3.3.0 by #229?

**Comment by blag** [comment]
Yes, thank you. Closing. 😄
